subsys/infinispan/src/main/resources/infinispan-cluster.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="urn:infinispan:config:9.4 http://www.infinispan.org/schemas/infinispan-config-9.4.xsd
+                                urn:infinispan:config:store:jdbc:9.4 http://www.infinispan.org/schemas/infinispan-cachestore-jdbc-config-9.4.xsd"
+            xmlns="urn:infinispan:config:9.4"
+            xmlns:jdbc="urn:infinispan:config:store:jdbc:9.4">
+
+  <cache-container default-cache="local" name="IndyCacheManager" shutdown-hook="DEFAULT" statistics="true">
+    <transport cluster="indy-cache-cluster"/>
+
+    <!-- distributed caches configuration -->
+    <distributed-cache-configuration name="cluster-template" statistics="true">
+      <memory>
+        <object size="200000" strategy="REMOVE" />
+      </memory>
+    </distributed-cache-configuration>
+
+    <distributed-cache name="cluster" configuration="cluster-template"/>
+
+    <distributed-cache name="schedule-expire-cache" configuration="cluster-template">
+      <expiration interval="300"/>
+      <persistence passivation="true">
+        <file-store shared="true" preload="false" fetch-state="true" path="${indy.data}/scheduler"/>
+      </persistence>
+    </distributed-cache>
+
+    <distributed-cache name="schedule-event-lock-cache" configuration="cluster-template">
+      <expiration interval="300"/>
+      <persistence passivation="true">
+        <file-store shared="true" preload="false" fetch-state="true" path="${indy.data}/scheduler-event-lock"/>
```

Comment:
We can not use file-based persistence for the clustered cache. We can just not persist it. This lock cache is meant to be holding ephemeral entries IIUIC.